src/base-commands/twilio-client-command.js

```diff
@@ -2,13 +2,14 @@ const chalk = require('chalk');
 const { flags } = require('@oclif/command');
 const BaseCommand = require('./base-command');
 const CliRequestClient = require('../services/cli-http-client');
-const { TwilioApiClient } = require('../services/twilio-api');
+const { TwilioApiClient, TwilioApiFlags } = require('../services/twilio-api');
 const { TwilioCliError } = require('../services/error');
-const { camelCase } = require('../services/naming-conventions');
+const { translateValues } = require('../services/javascript-utilities');
+const { camelCase, kebabCase } = require('../services/naming-conventions');
 const { HELP_ENVIRONMENT_VARIABLES } = require('../services/messaging/help-messages');
 
-// 'account-sid' is a special snowflake
-const ACCOUNT_SID = 'account-sid';
+// CLI flags are kebab-cased, whereas API flags are PascalCased.
```

Comment:
case implementation inherited from oclif and open API respectively?

Reply:
CLI case is by convention.

API case is specific to the Twilio API, and thus part of our OpenAPI spec.